### PR TITLE
Feat/export import tx final application state

### PR DESCRIPTION
**Pull request by vncoelho**
Deals with https://github.com/neo-project/neo/issues/712 during export and import only.

Under development, this is the initial draft for export, not yet fully tested.

### CODE REVIEW

File: ImportBlocks/ImportBlocks.cs [hunk 1]
```diff
@@ -94,6 +118,46 @@ private bool OnExport(string[] args)
...
+                _params.Add(txHash);
+                JObject getAppLogRequest = new JObject();
+                getAppLogRequest["id"] = -1;
+                getAppLogRequest["method"] = "getapplicationlog";
+                getAppLogRequest["params"] = _params;
+                JObject txState = System.RpcServer.ProcessRequest(null, getAppLogRequest);
+
+                Console.Write($"Result is {txState["result"]}\n");
+                if (!txState["result"])
+                    if ((int)txState["code"].AsNumber() == -100 || (int)txState["code"].AsNumber() == -32602 || (int)txState["code"].AsNumber() == -32600)
```
Comment (vncoelho) [inline ImportBlocks/ImportBlocks.cs]: @erikzhang, @shargon, it would be nice to change this line for any kind of error.

**Comment by vncoelho** [comment]
@erikzhang, what solution do you propose?

**Comment by erikzhang** [comment]
My solution is to manually copy the ApplicationLogs directory, and then the `ImportBlocks` plugin directly reads the json files.

**Comment by vncoelho** [comment]
We thought about that, @erikzhang, the current Application Logs folder for Mainnet at height 3687369 is 3.9G. 
What do you think, @igormcoelho?

That would be the easiest way without making changes.
Furthermore, with the MPT this might be unnecessary, right?

**Comment by vncoelho** [comment]
I think there is not reason to push this for 2.x anymore.

In addition, for 3.0 we gonna have better solutions.

**Pull request closed without merging**